x-pack/plugins/apm/common/alert_types.ts

```diff
@@ -45,6 +47,7 @@ export const ALERT_TYPES_CONFIG = {
     }),
     actionGroups: [THRESHOLD_MET_GROUP],
     defaultActionGroupId: 'threshold_met',
+    minimumLicenseRequired: 'basic',
```

Comment:
This needs a platinum license I think, as it is based on Machine Learning. @sqren?